reference-architecture/azure-ansible/allinone.json

```diff
@@ -0,0 +1,670 @@
+      }
+    },
+    {
+      "type" : "Microsoft.Storage/storageAccounts",
+      "name" : "[variables('StorageAccountName')]",
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[variables('location')]",
+      "tags" : {
+        "displayName" : "StorageAccount"
+      },
+      "properties" : {
+        "accountType" : "[variables('vmSizesMap')[parameters('vmSize')].storageAccountType]"
+      }
+    },
+    {
+      "apiVersion" : "2017-03-30",
+      "type" : "Microsoft.Compute/virtualMachines",
+      "name" : "[resourceGroup().name]",
+      "location" : "[resourceGroup().location]",
+      "dependsOn" : [
+        "[resourceId('Microsoft.Storage/storageAccounts/', variables('StorageAccountName'))]",
+        "[resourceId('Microsoft.Network/networkInterfaces/', variables('nicName'))]"
+      ],
+      "properties" : {
+        "hardwareProfile" : {
+          "vmSize" : "[parameters('vmSize')]"
+        },
+        "osProfile" : {
+          "computerName" : "[resourceGroup().name]",
+          "adminUsername" : "[parameters('adminUsername')]",
+          "adminPassword" : "[parameters('adminPassword')]",
+          "linuxConfiguration" : {
+            "disablePasswordAuthentication" : "true",
+            "ssh" : {
+              "publicKeys" : [
+                {
+                  "path" : "[variables('sshKeyPath')]",
+                  "keyData" : "[parameters('sshKeyData')]"
+                }
+              ]
+            }
+          }
+        },
+        "storageProfile" : {
+          "imageReference" : "[variables('rhel')]",
+          "osDisk" : {
+            "caching" : "ReadWrite",
+            "vhd" : {
+              "uri" : "[concat('http://', variables('StorageAccountName'), '.blob.core.windows.net/openshiftvmachines/', resourceGroup().name, '.vhd')]"
+            },
+            "name" : "osdisk",
+            "createOption" : "FromImage"
+          },
+          "dataDisks" : [
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "128",
+              "lun" : 0,
+              "name" : "datadisk0",
+              "vhd" : {
+                "uri" : "[concat('http://', variables('StorageAccountName'), '.blob.core.windows.net/openshiftvmachines/', resourceGroup().name, '_datadisk0.vhd')]"
+              }
+            },
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "128",
+              "lun" : 1,
+              "name" : "datadisk1",
+              "vhd" : {
+                "uri" : "[concat('http://', variables('StorageAccountName'), '.blob.core.windows.net/openshiftvmachines/', resourceGroup().name, '_datadisk1.vhd')]"
+              }
+            },
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "512",
```

Comment:
actually not sure it saves any money on the 32 as its such a small size, and 
128 is nothing for app data. I will delete the "spare disk if you really want, but think the value for developer gets tossed with it as well.
I'm fighting that in my packet.net, it really fills up fast.